docs/personal-data.md

```diff
@@ -223,9 +259,22 @@ A list of all the [Identity events]((../src/Surfnet/Stepup/Identity/Event/) in s
 - second_factor_type
 - preferred_locale
 - Forgettable: email
-- Forgettable: commonName
-- Forgettable: secondFactorIdentifier
-- Forgettable: vettingType (each vetting type has different data)
+- Forgettable: common_name
+- Forgettable: U2F (is second_factor_identifier)
+- Forgettable: vetting_type (each vetting type has different data)
+
+[U2fDevicePossessionProvenAndVerifiedEvent](../src/Surfnet/Stepup/Identity/Event/U2fDevicePossessionProvenAndVerifiedEvent.php)
+- identity_id
+- identity_institution
+- second_factor_id
+- registration_requested_at
+- preferred_locale
+- second_factor_type
+- Forgettable: U2F (is second_factor_identifier)
+- Forgettable: email
+- Forgettable: common_name
+  [U2fDevicePossessionProvenEvent](../src/Surfnet/Stepup/Identity/Event/U2fDevicePossessionProvenEvent.php)
```

Comment:
You forgot to fill in the details of this event?

Idea: mark all events that are deprecated in this document too. For example we no longer play new U2fDevicePossessionProvenEvent from our aggregates. But they are still supported as they are in the event stream. That is relevant data for this readme in my opinion.